File: Problem_4/Problem_4.py
class Solution:
    def longestCommonPrefix(self, strs: List[str]) -> str:
      
        res = ""
        strs.sort()
        first = strs[0];
        last = strs[-1];
        for i in range(0,min(len(first),len(last))):
            if first[i]!=last[i]:
                return res;
            
            res+=first[i];
        
        return res;
        

File: Problem_4/Problem_4.cs
public class Solution {
    public string LongestCommonPrefix(string[] strs) {
                StringBuilder res = new StringBuilder();
        Array.Sort(strs);
        String first = strs[0];
        String last = strs[strs.Length - 1];
        for(int i=0;i<Math.Min(first.Length,last.Length);i++){
            if(first[i]!=last[i]){
                return res.ToString();
            }
            res.Append(first[i]);
        }
        return res.ToString();
    }
}
